### ISSUE TITLE
Set up redirect URI automatically

### ISSUE THREAD

**Issue by singpolyma** [jjbohn/omniauth-openid-connect#47]
Based on omniauth callback routes, etc, like other strategies

**Comment by puregold117** [comment]
Good point, @singpolyma. It's better we have a documentation on overview describing how to set omniauth routes(authorize, callback).
